tailcfg/tailcfg.go

```diff
@@ -235,6 +235,10 @@ type Node struct {
 	ComputedName            string `json:",omitempty"` // MagicDNS base name (for normal non-shared-in nodes), FQDN (without trailing dot, for shared-in nodes), or Hostname (if no MagicDNS)
 	computedHostIfDifferent string // hostname, if different than ComputedName, otherwise empty
 	ComputedNameWithHost    string `json:",omitempty"` // either "ComputedName" or "ComputedName (computedHostIfDifferent)", if computedHostIfDifferent is set
+
+	// DataPlaneAuditLogID is the per-node logtail ID used for data plane audit logging.
+	// It is non-empty if and only if CapabilityDataPlaneAuditLogs is set.
```

Comment:
this "if and only if" bit isn't true as of this commit.

It's never set regardless :)

Reply:
Well, it is in the sense that no tailnets have that feature flag turned on, so even though they're not tied in yet in corp, this still holds. But if you feel strongly enough about it... :-)

Reply:
Yeah, but then your comment is making a promise that's a function of time rather than the code.

It means if in 3 months (when control has moved on) we sync oss back to this point in time and read the code, its promise in the client no longer holds.